linkedin/datahub
# [WIP] deps(resolving outdated dependencies): adding resolutions to upgrade packages

These dependencies have had issues raised about them by Dependabot. Adding resolutions to newer versions so they dont give us trouble.

Test Plan:
- start up Datahub
- Visit search, browse, dataset, chart, lineage views
- Write a tag to an entity. Verify the cache is properly updated. Refresh and verify it was persisted to storage.
- Update an entity's description w/ markdown. Verify the cache & storage are properly updated.

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Docs related to the changes have been added/updated (if applicable)
